flake.nix

```diff
@@ -157,8 +157,9 @@
       common-gpu-amd-sea-islands = import ./common/gpu/amd/sea-islands;
       common-gpu-amd-southern-islands = import ./common/gpu/amd/southern-islands;
       common-gpu-intel = import ./common/gpu/intel.nix;
-      common-gpu-nvidia = import ./common/gpu/nvidia.nix;
-      common-gpu-nvidia-disable = import ./common/gpu/nvidia-disable.nix;
+      common-gpu-nvidia = import ./common/gpu/nvidia/prime.nix;
+      common-gpu-nvidia-nonprime = import ./common/gpu/nvidia;
```

Comment:
In general LGTM, but I think we need to bikeshed this name a little. However I can also not think about a better name, so for now I am going to approve.

But if someone can think about a better name, just put it here.

Reply:
For reference, the vendor terms would be `-offload` and `-output`, but those terms are confusing to anyone without prior knowledge about prime, so I think mentioning prime is important - that way a user with no knowledge knows what to read up on.

Just having `-output` would be very confusing anyway.

`-desktop` and `-laptop` would mimick existing terminology, but is confusing for anyone actually using their iGPU.

I'd like for it to be swapped, and have a `-prime` instead of `-nonprime`, especially since this mimics the module names, but that can't be done without a breaking change.

Ultimately I left it like this because it's the best I can come up with.